settings.py-dist

```diff
@@ -1,16 +1,30 @@
 # Django settings for desktop_signing_webapp project.
 
 import os
-import ldap
-from django_auth_ldap.config import LDAPSearch, GroupOfNamesType
+
+from settings_non_public import * # Either "settings_non_public" or "settings_public"
```

Comment:
Maybe instead of having different settings files for the two instances, use environment variables and set them in the apache virtual hosts